src/data.h

```diff
@@ -25,92 +25,109 @@
 
 #pragma once
 
+/*
+ * For settings, the `where` field should normally be '0'.
+ * systems are assumed to be a server. If you need to specify
+ * a setting for desktop only, use '1'.
+ * If you need to have an alternate setting for server and desktop,
+ * make 2 entries with '1' and '-1'.
```

Comment:
This seems a little confusing when the above instruction says use 1 for desktop only, was that supposed to be -1? And 1 is server only?

Maybe consider default 0 (doesn't care about platform/cpuinfo), 1 is server specific, 2 desktop, (3 etc as more platform groups make sense).

Reply:
typo, I need to fix the numbers here.